test/validators/no-returns-required/frequency-collected.validator.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const FrequencyCollectedValidator = require('../../../app/validators/return-requirements/frequency-collected.validator.js')
+
+describe('Frequency Collected validator', () => {
+  describe('when valid data is provided', () => {
+    const payload = {
+      frequencyCollected: 'monthly'
+    }
+
+    it('confirms the data is valid', async () => {
+      const result = FrequencyCollectedValidator.go(payload)
+
+      expect(result.value.frequencyCollected).to.exist()
```

Comment:
How does the property frequencyCollected existing confirm that the data is valid? If you add this same expect block to the test below 'when invalid data is provided' the text still passes as frequencyCollected still exists. Is there a better way this could be tested?